### DISCUSSION TITLE
Resizing window moves cursor

### DISCUSSION THREAD

**Discussion by Varal7** [LazyVim/LazyVim#5245] [Q&A] [answered]
https://github.com/user-attachments/assets/28544884-d467-4293-a292-12d7616341bf

When I resize my nvim window, it changes the position of my cursor (nvim doesn't have this behaviour by default).

Seems related to https://github.com/LazyVim/LazyVim/discussions/2451#discussioncomment-9939179
But I can't find the autocommand that @xzbdmw mentioned in that thread

**Comment by dpetka2001** [comment, marked as answer]
Put in your `options.lua` file `vim.opt.splitkeep = "cursor"`. For more info `:h 'splitkeep'`
